Chaincode does not start due to missing dependency in fabric-shim

Container logs:
```+ CHAINCODE_DIR=/usr/local/src
+ cd /usr/local/src
+ npm start -- --peer.address peer0.hospital.healthcert.io:7052

> healthcert-contract@1.0.0 start
> fabric-chaincode-node start --peer.address peer0.hospital.healthcert.io:7052

node:internal/modules/cjs/loader:998
  throw err;
  ^

Error: Cannot find module 'fast-safe-stringify'
Require stack:
- /usr/local/src/node_modules/fabric-shim/lib/logger.js
- /usr/local/src/node_modules/fabric-shim/cli.js
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:995:15)
    at Function.Module._load (node:internal/modules/cjs/loader:841:27)
    at Module.require (node:internal/modules/cjs/loader:1067:19)
    at require (node:internal/modules/cjs/helpers:103:18)
    at Object.<anonymous> (/usr/local/src/node_modules/fabric-shim/lib/logger.js:12:23)
    at Module._compile (node:internal/modules/cjs/loader:1165:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1219:10)
    at Module.load (node:internal/modules/cjs/loader:1043:32)
    at Function.Module._load (node:internal/modules/cjs/loader:878:12)
    at Module.require (node:internal/modules/cjs/loader:1067:19) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/usr/local/src/node_modules/fabric-shim/lib/logger.js',
    '/usr/local/src/node_modules/fabric-shim/cli.js'
  ]
```

Dependencies of the contract:
```
  "dependencies": {
    "fabric-contract-api": "^2.5.2",
    "fabric-shim": "^2.5.2"
  },
  "devDependencies": {
    "tslint": "^6.1.3",
    "typescript": "^4.9.5"
  }
```

I believe this is because the `fast-safe-stringify` package is used in `fabric-shim` but is not included in it's `package.json`. It is only included as a dependency of `fabric-contract-api`.

https://github.com/hyperledger/fabric-chaincode-node/blob/0d446fd5ec29d8cd88820461a0b3db6831ff65ec/libraries/fabric-shim/lib/logger.js#L12

--- a/libraries/fabric-shim/package.json
+++ b/libraries/fabric-shim/package.json
@@ -62,6 +62,7 @@
     "ajv": "^6.12.2",
     "fabric-contract-api": "3.0.0-unstable",
     "fabric-shim-api": "3.0.0-unstable",
+    "fast-safe-stringify": "^2.1.1",
     "fs-extra": "^10.0.1",
     "reflect-metadata": "^0.1.13",
     "winston": "^3.7.2",

